Google.Api.Generator/Google.Api.Generator.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="Google.LongRunning" Version="2.0.0-beta01" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.10.0" />
     <PackageReference Include="Moq" Version="4.12.0" />
+    <PackageReference Include="SharpYaml" Version="1.6.6" />
```

Comment:
Is there a reason for picking SharpYaml here? We're already using YamlDotNet elsewhere, and I think it would be better to use a single implementation across multiple tools if possible. That said, if SharpYaml has features that YamlDotNet doesn't (and that you need) that's fine.